openadapt/visualize1.py

```diff
@@ -0,0 +1,209 @@
+from pprint import pformat
+
+from loguru import logger
+from nicegui import ui
+from tqdm import tqdm
+
+from openadapt import config
+from openadapt.crud import get_latest_recording
+from openadapt.events import get_events
+from openadapt.utils import (
+    EMPTY,
+    configure_logging,
+    display_event,
+    image2utf8,
+    row2dict,
+    rows2dicts,
+    plot_performance,
+)
+
+SCRUB = config.SCRUB_ENABLED
+if SCRUB:
+    # too many warnings from scrubbing
+    __import__("warnings").filterwarnings("ignore", category=DeprecationWarning)
+    from openadapt import scrub
+
+
+LOG_LEVEL = "INFO"
+MAX_EVENTS = None
+MAX_TABLE_CHILDREN = 5
+PROCESS_EVENTS = True
+
+
+def create_tree(tree_dict, max_children=MAX_TABLE_CHILDREN):
+    tree_data = []
+    for key, value in tree_dict.items():
+        if value in EMPTY:
+            continue
+        node = {
+            "id": str(key)
+            + f"{(': '  + str(value)) if not isinstance(value, (dict, list)) else ''}"
```

Comment:
Seems like the f-string is unused here. Can you please re-write this to use the f-string?